readme.md

```diff
@@ -77,6 +77,10 @@ A universal method to calculate [mouse sensitivity](#sensitivity), literally ref
 
 Also known as: `CM/360`
 
+## Crosshair Placement
+
+The ability and the process to put and acquire the correct spot for the crosshair to be situated/placed for a certain/specific situation often to maximize damage uptime/time to damage.
```

Comment:
Suggestion: remove <blank>/<blank> references.  Maybe rewrite as: The process to acquire the optimal location for one's crosshair to be placed prior to a gunfight in order to maximize damage uptime after initiating gunfire.

Reply:
idk about placement prior because crosshair placement can include things like headshots, going for the feet for movement tracking, etc.. so can make it a bit more succinct but I don't think there should be a significant change to the context

Reply:
i will cut it down a little bit with the suggestions that don't conflict with my comment above